wagtail/admin/tests/pages/test_explorer_view.py

```diff
@@ -403,9 +403,9 @@ def test_breadcrumb_uses_specific_titles(self):
 
         expected = (
             """
-            <li class="w-h-full w-flex w-items-center w-overflow-hidden w-transition w-duration-300 w-whitespace-nowrap w-flex-shrink-0 w-font-bold w-max-w-0" data-breadcrumb-item hidden>
```

Comment:
@PaarthAgarwal  I think the problem is this change here - you have removed `w-max-w-0` when trying to fix the tests but this change has made the failing tests more confusing.

Instead just add a single space after because of the additional logic.

```
             <li class="w-h-full w-flex w-items-center w-overflow-hidden w-transition w-duration-300 w-whitespace-nowrap w-flex-shrink-0 w-font-bold w-max-w-0 " data-breadcrumb-item hidden>
```